wandb/sdk/launch/sweeps/scheduler.py

```diff
@@ -153,6 +166,11 @@ def run(self) -> None:
                     break
                 self._update_run_states()
                 self._run()
+                # if we hit the run_cap, now set to stopped after launching runs
+                if self.state == SchedulerState.FLUSH_RUNS:
+                    if len(self._runs.keys()) == 0:
+                        wandb.termlog(f"{LOG_PREFIX}Done polling on runs, exiting.")
+                        self.state = SchedulerState.STOPPED
```

Comment:
COMPLETE?

Reply:
Also, call self.exit()?

Reply:
I'm pretty sure self.exit() gets called from the else clause below. `SchedulerState.STOPPED` because (at least currently) the only time we are in this condition we have been stopped by hitting the `run_cap`. Happy to change to finished.